Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/client/CONNECTCXFClientSecured.java

```diff
@@ -43,18 +43,17 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(CONNECTCXFClientSecured.class);
 
-    CONNECTCXFClientSecured(ServicePortDescriptor<T> portDescriptor, String url, AssertionType assertion,
-            String wsAddressingTo, String SoapHeader) {
+    CONNECTCXFClientSecured(ServicePortDescriptor<T> portDescriptor, String url, AssertionType assertion) {
         super(portDescriptor, url, assertion, new CachingCXFSecuredServicePortBuilder<>(portDescriptor));
-        decorateEndpoint(assertion, wsAddressingTo, portDescriptor.getWSAddressingAction(), SoapHeader, null, null);
+        decorateEndpoint(assertion, url, portDescriptor.getWSAddressingAction(), null, null);
```

Comment:
I removed an unused method in this class which I noticed while doing this pr but it's out of scope of this task and I'd prefer to not do any more refactoring here.